inductiva/tasks/task.py

```diff
@@ -216,6 +236,11 @@ def wait(self, polling_period: int = 5) -> models.TaskStatusCode:
                         "while performing the task.", status)
             prev_status = status
 
+            if (self._tasks_ahead is not None and
+                    self._tasks_ahead != prev_tasks_ahead):
+                logging.info("Number of tasks ahead of task %s: %s", self.id,
+                             self._tasks_ahead)
```

Comment:
this is a bit verbose and, in my opinion, will pollute stdout quite a lot, specially if it has to wait several minutes in the queue. In the worst case, and if the task only moves  by one slot every 5 seconds, then I foresee several of these lines being printed. If this was a dynamic print, then that problem would be solved because the line would be re-written every time. But this is using the logging mechanism!

However, we can print directly to stdout and have that feedback given to the user. Something like:

```python
while ...:
    printf("\33[2K\r", file=sys.stdout) <-- because numbers decrease and we want the buffer to be cleared out
    print("Number of tasks ahead of task %s: %s", self.id, self._tasks_ahead, file=...)
sys.stdout.flush() <-- make sure logs are flushed
printf("\33[2K\r") <-- a final cleanup
```